jupyter-book/conditions/perturbation_modeling.ipynb

```diff
@@ -22,12 +22,12 @@
    "metadata": {},
```

Comment:
<p>Are these treatments causing the immune checkpoints respectively, or not?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/cross-modal-single-cell-best-practices/pull/91/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='108'/>

Reply:
<p>quote paper:</p><pre class="ql-syntax" spellcheck="false">We found that stimulating THP-1 cells with a combination of IFN-γ, decitabine (DAC) and transforming growth factor (TGF)-β1 resulted in robust induction of three immune checkpoints: PD-L1, PD-L2 and CD86 (Supplementary Fig. 1a).
<code></code>`
</pre>